tests/IntegrationTest.php

```diff
@@ -101,6 +101,10 @@ public function gettingPlaintextStuffFromEncryptedGoogleShouldNotWork()
 
     public function testConnectingFailsIfConnectorUsesInvalidDnsResolverAddress()
     {
+        if (PHP_OS === 'Darwin') {
+            $this->markTestSkipped('Skipped on macOS due to a bug in reactphp/dns (solved in reactphp/dns#171)');
```

Comment:
LGTM. Adding a link here only for reference: https://github.com/reactphp/dns/pull/171

Reply:
For the reference: We might as well update this now that https://github.com/reactphp/dns/pull/171 has been released a month ago. I'm good with this either way. :+1: